movieapp/src/containers/AddEditMovieForm/AddEditMovieForm.tsx

```diff
@@ -0,0 +1,79 @@
+import {Button} from "../../components/Button/Button";
+import {useTranslation} from "react-i18next";
+
+type MovieProps = {
+    title: string | '';
+    description: string | '';
+    key: string | '';
+}
+
+type MovieFormProps = {
+    onCloseModal: Function;
+    onAddMovie: Function;
+    currentValue:MovieProps;
```

Comment:
space after : 

eslint should handle it

Reply:
Fixed